src/Char.mo

```diff
@@ -2,22 +2,40 @@
 import Prim "mo:prim";
 module {
 
-  /// Convert a character to a word.
-  public let toWord32 : Char -> Word32 = Prim.charToWord32;
+  /// Convert character `c` to a word containing its Unicode scalar value.
+  public let toWord32 : (c : Char) -> Word32 = Prim.charToWord32;
```

Comment:
Drive-by comment: I wonder why we have toWord32 but not toNat or toNat32. Wouldn't those be useful as well?